ddht/v5_1/constants.py

```diff
@@ -28,13 +28,23 @@
 )
 
 
+DEFAULT_BOOTNODES: Tuple[str, ...] = (
+    # 94.63.192.187:30303
+    # "enr:-IS4QMKRqKacvGacwsChKkhY9OWyRAJvCkUG3mTlM-9DOtqcS7Zr7vVkf6s20ju6wCMNFE_ivWwYYp1CqA_ybJ9R_O8DgmlkgnY0gmlwhF4_wLuJc2VjcDI1NmsxoQLhZcNMi9TtZ9RxmjGNVG83YFyWQnyVdKr74_qXl7mbb4N1ZHCCdl8",  # noqa: E501
+    # 192.168.1.112:30303
+    "enr:-IS4QAVuwe64nAc-Lki-UHRUdpHzAB5xX_Uff0aQDBjIXgMlS7U7WjJ0h0Gja3-YTTTMTt6S5GG0doViOVYPMpYb2goDgmlkgnY0gmlwhMCoAXCJc2VjcDI1NmsxoQLhZcNMi9TtZ9RxmjGNVG83YFyWQnyVdKr74_qXl7mbb4N1ZHCCdl8",  # noqa: E501
+)
+
+
 PACKET_VERSION_1 = b"\x00\x01"
 
 ID_NONCE_SIGNATURE_PREFIX = b"discovery v5 identity proof"
 
 HEADER_PACKET_SIZE = 23
 
-PROTOCOL_ID = b"discv5"
+# PROTOCOL_ID = b"discv5"
+# XXX: Must be the same length as "discv5"
+PROTOCOL_ID = b"notdv5"
```

Comment:
Brainstorm:
- "ethdat"
- "estate"
- ~"ethp2p"~
- "datnet"

Reply:
this is a leftover from guilherme's edits, we can leave it as `discv5`.  Nothing wrong with being part of the main network.